torch/testing/_internal/composite_compliance.py

```diff
@@ -197,38 +196,36 @@ def wrap(e):
                         'regular Tensor but the other tensors are Tensor Subclasses. '
                         'Please try to avoid this in-place operation.')
 
-            with enable_reentrant_dispatch():
-                with contextlib.nullcontext() if enable_recursive_torch_dispatch else no_dispatch():
-                    unwrapped_args = tree_map(unwrap, args)
-                    unwrapped_kwargs = tree_map(unwrap, kwargs)
-                    unwrapped_rs = func(*unwrapped_args, **unwrapped_kwargs)
-                    rs = tree_map(wrap, unwrapped_rs)
+            with contextlib.nullcontext() if enable_recursive_torch_dispatch else no_dispatch():
```

Comment:
Could it be possible to remove the enable_recursive_torch_dispatch too? It probably doesn't matter to leave it on either for non-mode python dispatch.

Reply:
I'll test that on a separate PR